### PR TITLE
styles(destroy): iconify destroy button

### PR DESCRIPTION
**Pull request by johngruen**
@nchapman @pdehaan: this directly addresses #280, but it's also setting up formally styling the .actions area. This PR isn't intended to totally match @bryanbell's mockups, just an incremental improvement.

### CODE REVIEW

File: app/styles/modules/_actions.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      @include hidpi-background-image(icon-delete-active, 20px 20px);
+    }
+  }
+}
+
+.medium .actions,
+.large .actions {
+  clear: both;
+  display: block;
+}
```
Comment (pdehaan) [inline app/styles/modules/_actions.scss]: FWIW, I think you can tweak this to (or not, your call):

```css
.actions {
  ...

  .medium &,
  .large & {
    clear: both;
    display: block;
  }
}
```

File: app/styles/modules/_user_page.scss [hunk 2]
```diff
@@ -3,8 +3,8 @@
   border: 1px solid $baby-grey;
   // negative spread keeps the shadow from leaking out the sides
   box-shadow: 0 2px 4px -3px rgba(0, 0, 0, 0.14);
-  overflow: auto;
   padding: $double-gap;
+  overflow: auto;
```
Comment (pdehaan) [inline app/styles/modules/_user_page.scss]: A-Z here and in `.small` and in the `&:after` on L122.

**Comment by pdehaan** [comment]
Looks good locally!
Curious, why not use an SVG font like http://fortawesome.github.io/Font-Awesome/icon/trash-o/ or https://icomoon.io/#linearicons

**Comment by nchapman** [comment]
> why not use an SVG font

SVG/font icons are usually not sharp at small sizes in standard def. It's also a lot more trouble to have to translate their visual style into CSS rather than just exporting new versions as the style evolves.

**Comment by pdehaan** [comment]
Oh weird, I thought we switched to Fontello for FxA content server:
[**mozilla/fxa-content-server** /app/styles/fontello.css](https://github.com/mozilla/fxa-content-server/blob/master/app/styles/fontello.css) (see https://github.com/mozilla/fxa-content-server/pull/1405)

TL;DR

**Comment by nchapman** [comment]
I'll just leave this here: https://github.com/mozilla/fxa-content-server/blob/master/app/styles/fontello.css#L11. If one cares about pixel perfection (we do), fonts are hard.

**Comment by johngruen** [comment]
@pdehaan @nchapman: we should at least spritify these things eventually, once we get the final assets

**Comment by nchapman** [comment]
> we should at least spritify these things

@johngruen I talked to @bryanbell about that last night. He's on it.

**Comment by nchapman** [comment]
:+1:

**Comment by pdehaan** [comment]
Saw this on the weekend: https://www.npmjs.com/package/grunt-spritesmith
Not sure if having Grunt maintain our spritesheet would be overkill.
